### PR TITLE
chafa: fix cross-compilation

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by amjoseph-nixpkgs** [comment]
@ofborg build pkgsCross.aarch64-multiplatform.chafa

File: pkgs/tools/misc/chafa/default.nix [hunk 1]
```diff
@@ -31,7 +32,9 @@ stdenv.mkDerivation rec {
   patches = [ ./xmlcatalog_patch.patch ];
 
   preConfigure = ''
-    ./autogen.sh
+    substituteInPlace ./autogen.sh --replace pkg-config '$PKGCONFIG'
```
Comment (NickCao) [inline pkgs/tools/misc/chafa/default.nix:35]: Or `$PKG_CONFIG`? Which is set by stdenv.

**Review by Artturin — APPROVED** [review]
tested after applying https://github.com/NixOS/nixpkgs/pull/225360

**Comment by amjoseph-nixpkgs** [comment]
(Re)built on:
- [x] `x86_64-linux` (native)
- [x] `aarch64-linux` (cross from `x86_64-linux`)

**Pull request merged**